spec/internal/app/controllers/posts_controller.rb

```diff
@@ -11,6 +12,12 @@ class PostsController < Matterhorn::Base
 
   add_env :current_user
 
+  paginates_with Matterhorn::Paging::Default
+
+  # used for inclusions
+  allow_collection_params \
```

Comment:
I get some failling specs when removing https://github.com/blakechambers/matterhorn/pull/69/files#diff-1f778ea8ff9d56261674bf6ecde3f6c0R23 

 		Failures:

		  1) show with request: 'GET /posts/:id.json' when defining a custom scope should included scoped votes
		     Failure/Error: expect(body[:includes].execute.count).to eq(1)
       
		       expected: 1
		            got: 0
       
		       (compared using ==)
		     # ./spec/resources_api/show_spec.rb:53:in `block (4 levels) in <top (required)>'

		  2) show with request: 'GET /posts/:id.json' when defining a custom scope should include scoped authors
		     Failure/Error: expect(body[:includes].execute.count).to eq(1)
       
		       expected: 1
		            got: 0
       
		       (compared using ==)
		     # ./spec/resources_api/show_spec.rb:61:in `block (4 levels) in <top (required)>'

		  3) index with request: 'GET /posts.json' when requesting inclusions should included scoped votes
		     Failure/Error: expect(body[:includes]).to include_a_provided(users_vote)
		       expected #<SerialSpec::ParsedBody:0x00000101dbb508 @selector=[[:[], :includes]], @raw_body="{\"data\":[{\"_id\":\"5547c61e6e6576bbd55e0000\",\"type\":\"post\",\"links\":{\"author\":{\"linkage\":{\"id\":\"5547c61e6e6576bbd55d0000\",\"type\":\"users\"},\"related\":\"http://example.org/users/5547c61e6e6576bbd55d0000?include=vote\"},\"vote\":{\"linkage\":{\"post_id\":\"5547c61e6e6576bbd55e0000\",\"type\":\"votes\"},\"related\":\"http://example.org/posts/5547c61e6e6576bbd55e0000/vote?include=vote\"},\"comments\":{\"linkage\":{\"post_id\":\"5547c61e6e6576bbd55e0000\",\"type\":\"comments\"},\"related\":\"http://example.org/posts/5547c61e6e6576bbd55e0000/comments?include=vote\"},\"self\":\"http://example.org/posts/5547c61e6e6576bbd55e0000?include=vote\"},\"author_id\":\"5547c61e6e6576bbd55d0000\",\"body\":\"body\",\"initial_comments_ids\":null}],\"links\":{\"author\":\"http://example.org/users/{posts.author_id}?include=vote\",\"vote\":\"http://example.org/posts/{posts._id}/vote?include=vote\",\"comments\":\"http://example.org/posts/{posts._id}/comments?include=vote\",\"self\":\"http://example.org/posts/{posts._id}?include=vote\",\"next\":\"http://example.org/posts?include=vote\\u0026limit=25\\u0026offset=25\",\"prev\":\"http://example.org/posts?include=vote\\u0026limit=25\",\"first\":\"http://example.org/posts?include=vote\\u0026limit=25\"},\"orders\":{\"recent\":\"http://example.org/posts?include=vote\\u0026order=recent\",\"oldest\":\"http://example.org/posts?include=vote\\u0026order=oldest\"},\"includes\":[]}"> to include a provided #<Vote _id: 5547c61e6e6576bbd5600000, score: 1, user_id: BSON::ObjectId('5547c61e6e6576bbd55d0000'), post_id: BSON::ObjectId('5547c61e6e6576bbd55e0000')>
		     # ./spec/resources_api/index_spec.rb:73:in `block (4 levels) in <top (required)>'

		  4) index with request: 'GET /posts.json' when requesting inclusions should include scoped authors
		     Failure/Error: expect(body[:includes]).to include_a_provided(current_user)
		       expected #<SerialSpec::ParsedBody:0x00000101eea938 @selector=[[:[], :includes]], @raw_body="{\"data\":[{\"_id\":\"5547c61e6e6576bbd5640000\",\"type\":\"post\",\"links\":{\"author\":{\"linkage\":{\"id\":\"5547c61e6e6576bbd5630000\",\"type\":\"users\"},\"related\":\"http://example.org/users/5547c61e6e6576bbd5630000?include=author\"},\"vote\":{\"linkage\":{\"post_id\":\"5547c61e6e6576bbd5640000\",\"type\":\"votes\"},\"related\":\"http://example.org/posts/5547c61e6e6576bbd5640000/vote?include=author\"},\"comments\":{\"linkage\":{\"post_id\":\"5547c61e6e6576bbd5640000\",\"type\":\"comments\"},\"related\":\"http://example.org/posts/5547c61e6e6576bbd5640000/comments?include=author\"},\"self\":\"http://example.org/posts/5547c61e6e6576bbd5640000?include=author\"},\"author_id\":\"5547c61e6e6576bbd5630000\",\"body\":\"body\",\"initial_comments_ids\":null}],\"links\":{\"author\":\"http://example.org/users/{posts.author_id}?include=author\",\"vote\":\"http://example.org/posts/{posts._id}/vote?include=author\",\"comments\":\"http://example.org/posts/{posts._id}/comments?include=author\",\"self\":\"http://example.org/posts/{posts._id}?include=author\",\"next\":\"http://example.org/posts?include=author\\u0026limit=25\\u0026offset=25\",\"prev\":\"http://example.org/posts?include=author\\u0026limit=25\",\"first\":\"http://example.org/posts?include=author\\u0026limit=25\"},\"orders\":{\"recent\":\"http://example.org/posts?include=author\\u0026order=recent\",\"oldest\":\"http://example.org/posts?include=author\\u0026order=oldest\"},\"includes\":[]}"> to include a provided #<User _id: 5547c61e6e6576bbd5630000, name: "name", auth_token: "authenticate">
		     # ./spec/resources_api/index_spec.rb:80:in `block (4 levels) in <top (required)>'